src/renderer/src/components/Calendar.tsx

```diff
@@ -287,10 +289,10 @@ export const Calendar = ({
         range_middle:
           'bg-gray-100/10 !text-white [&>button]:!bg-transparent [&>button]:!text-white [&>button]:hover:!bg-transparent [&>button]:hover:!text-white',
         range_end: 'day-range-end rounded bg-neon [&>button]:bg-neon [&>button]:hover:neon [&>button]:hover:neon',
-        selected: '[&>button]:bg-neon [&>button]:text-asphalt [&>button]:hover:!bg-neon [&>button]:hover:text-asphalt',
+        selected: '[&>button]:!bg-neon [&>button]:!text-asphalt',
         today:
-          '[&>button]:bg-gray-300/10 [&>button]:data-[selected=true]:bg-neon [&>button]:data-[selected=true]:text-asphalt',
-        outside: 'day-outside  opacity-50',
+          '[&>button]:bg-gray-300/10 [&>button]:data-[selected=true]:bg-neon [&>button]:data-[selected=true]:[not(:disabled)]:text-asphalt',
+        outside: 'day-outside opacity-30',
```

Comment:
I wouldn't change the opacity, as it wouldn't be very difficult to differentiate what is disabled from what is outside